src/components/ens-registration/SearchResult/SearchResultGradientIndicator.tsx

```diff
@@ -28,19 +29,19 @@ const SearchResultGradientIndicator = ({
   switch (type) {
     case 'availability':
       if (isRegistered) {
-        text = '😭 Taken';
+        text = `😭 ${lang.t('profiles.search.taken')}`;
         gradient = colors.gradients.transparentToLightOrange;
       } else {
-        text = '🥳 Available';
+        text = `🥳 ${lang.t('profiles.search.available')}`;
         gradient = colors.gradients.transparentToGreen;
       }
       break;
     case 'expiration':
-      text = `Til ${expirationDate}`;
+      text = `${lang.t('profiles.search.until')} ${expirationDate}`;
```

Comment:
idk, but should we do interpolation here? there could be other languages that could have "til" on the right hand side... i guess it is good practice to interpolate string variables in i18n generally

```suggestion
      text = `${lang.t('profiles.search.until', { content: expirationDate })}`;
```